contribs/gmf/src/directives/partials/layertree.html

```diff
@@ -0,0 +1,17 @@
+<span ng-if="::!layertreeCtrl.isRoot"
+    ng-class="['depth-' + layertreeCtrl.depth, gmfLayertreeCtrl.getResolutionStyle(layertreeCtrl.node), gmfLayertreeCtrl.getNoSourceStyle(layertreeCtrl)]">
+  <a href ng-click="::gmfLayertreeCtrl.toggleActive(layertreeCtrl)">
+    <span class="state" ng-class="[gmfLayertreeCtrl.getSetActive(layertreeCtrl)]"></span>
```

Comment:
But in case of that's a leaf, I must get the 'state' style too... I don't undertsand :-/